client/src/adapter/dosDashboardRefresh.ts

```diff
@@ -25,10 +25,26 @@ function parseRounds(rounds: any[]) {
     return rounds.map(parseRound);
 }
 
-function parseDisCount(data: any): number {
+function parseDisagreementCount(data: any): number {
+    if (_.isEmpty(data)) {
+        return null;
+    }
+
     return _.sum(_.values(data));
 }
 
+function parseDiscrepancyCounts(data: any): any {
```

Comment:
I see that these had to be split into their own functions. Pity about the loss of the old function name. :)